shell/common/switches.cc

```diff
@@ -66,6 +66,8 @@ static const std::string gAllowedDartFlags[] = {
     "--write-service-info",
     "--null_assertions",
     "--strict_null_safety_checks",
+    "--enable-display-list",
```

Comment:
These flags are meant to be sent directly to the Dart VM during initialization. For engine specific flags, we add them to `shell/common/switches.h`

Reply:
My impression is that would advertise these switches. I was looking to keep them un-advertised for now...?

Reply:
All switches are un-advertised and unstable really.